modules/mux/targets/host/source/command_buffer.cpp

```diff
@@ -67,8 +67,9 @@ size_t calcPackedArgsAllocSize(
 // Iterates through the argument descriptors and for each argument sets the
 // location in the packed args location to the correct value.
 void populatePackedArgs(
-    uint8_t *packed_args_alloc,
+    mux::dynamic_array<uint8_t> &packed_args,
     const mux::dynamic_array<mux_descriptor_info_t> &descriptors) {
+  uint8_t *const packed_args_alloc = packed_args.begin();
```

Comment:
maybe data() is clearer (appreciate it's the same thing here).

Reply:
Agreed, and likewise elsewhere, will update.